lib/handlers/networkHandler.js

```diff
@@ -61,19 +61,23 @@ const responseHandler = response => {
   eventHandler.emit('responseReceived', response);
 };
 
+const getMatchingInterceptor = (interceptor, url) => {
```

Comment:
Sorry for the late review comment.
Can we rename this method to `clearIntercept`?

Reply:
@zabil I don't think the method name would be correct. It just identifies the matching interceptor

Reply:
I meant renaming `resetInterceptor` to `clearInterceptor`

Reply:
I renamed resetIntercept to clearIntercept if thats what you were referring to .